packages/core/src/lib/session.ts

```diff
@@ -12,3 +14,13 @@ export const assignInteractionResults = async (
   });
   ctx.body = { redirectTo };
 };
+
+export const saveApplicationId = async (userId: string, applicationId: string) => {
```

Comment:
`applicationId` sounds ambiguous. mind to change to a more clear name?

Reply:
Updated to `saveFirstConsentedApplicationId`. PTAL